recipes/tweak/meta.yaml

```
@@ -0,0 +1,36 @@
+{% set name = "tweak" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1636397466a127d054db391bd594efdc10ce7fa5eee8607b89520f8906361f37
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - tweak
+
+about:
+  home: https://github.com/kislyuk/tweak
+  license: Apache Software
```

This should follow the SPDX names (please check I've guessed the right version):
```suggestion
  license: Apache-2.0
```

---

Thanks @chrisburr, I'm new to conda-forge and this was just what was generated in the skeleton. I've checked and you're right (though I'm not the package author).